src/proto/Stellar.proto

```diff
@@ -26,28 +26,44 @@ message SigningInput {
 
     int64 sequence = 3;
 
-    string account = 4;
+    // Optional time bound from
+    int64 valid_after = 4;
 
-    string destination = 5;
+    // Optional time bound to
+    int64 valid_before = 5;
+
+    string account = 6;
+
+    string destination = 7;
+
```

Comment:
any chance we can move all the values into operation proto itself?

Reply:
Having separate proto entities for the operations is a good idea, only that it would not be backward compatible any more.

Reply:
it's ok to make this breaking change, as far as I know, no one uses this in SDK

Reply:
Done